centreon/www/front_src/src/ResourceAccessManagement/AddEditResourceAccessRule/FormInputs/styles/DeleteDatasetButton.styles.ts

```diff
@@ -0,0 +1,28 @@
+import { makeStyles } from 'tss-react/mui';
+
+export const useDeleteDatasetButtonStyles = makeStyles()((theme) => ({
+  deleteDatasetButtonContainer: {
+    alignItems: 'center',
+    borderBottom: `2px solid ${theme.palette.divider}`,
+    borderRight: `2px solid ${theme.palette.divider}`,
+    borderTop: `2px solid ${theme.palette.divider}`,
+    display: 'flex',
+    marginBottom: theme.spacing(7.25),
+    marginLeft: theme.spacing(4),
+    marginRight: theme.spacing(4),
+    marginTop: theme.spacing(1),
+    width: theme.spacing(0.75)
+  },
+  deleteIcon: {
+    color: theme.palette.common.white,
+    margin: '0px'
+  },
+  deleteIconChip: {
+    '&:hover': {
+      backgroundColor: theme.palette.chip.color.error
+    },
+    backgroundColor: theme.palette.divider,
+    position: 'relative',
+    right: theme.spacing(0.6)
```

Comment:
```suggestion
    right: theme.spacing(0.5)
```

Reply:
![image](https://github.com/centreon/centreon/assets/108675430/cc9806b7-7a43-48cd-80fe-4ee87b54abc3)

It allows to align the chip with delete button on the brace

Reply:
Done